config/routes.rb

```diff
@@ -0,0 +1,28 @@
+Rails.application.routes.draw do
+  devise_for :users
+
+  # Define the root path route ("/")
+  # root "articles#index"
+  resources :foods, only: [:index, :show, :new, :create, :destroy]
+
+  authenticated :user do
+    root 'recipes#index', as: :authenticated_root
+  end
+
+  root 'home#index'
+
```

Comment:
- [ ] I commend the extent you went after the call to make these changes, I also pulled it again but I realized that there is no index file in the home folder, I will encourage you to create it.